lib/trino-hive-formats/pom.xml

```diff
@@ -100,6 +100,12 @@
             <artifactId>modernizer-maven-annotations</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+            <scope>provided</scope>
```

Comment:
I think `runtime` is the correct scope for dependencies that are used in test code but are also needed transitively at runtime. `provided` scope makes it available on the compile classpath, but these shouldn't be on the compile classpath since they aren't referenced in production code.

Reply:
I think I tried that but it didn't work. Let me try again just in case I missed it.